lib/topological_inventory/openshift/collector.rb

```diff
@@ -4,18 +4,20 @@
 require "topological_inventory/openshift/logging"
 require "topological_inventory/openshift/connection"
 require "topological_inventory/openshift/parser"
+require "topological_inventory/openshift/collector/application_metrics"
```

Comment:
@carbonin should this be moved up here https://github.com/ManageIQ/topological_inventory-openshift/pull/80/files#diff-9e7a99bb1d56c71b44f0ba9aecf117a6R38 ?

Reply:
Yeah, I can move this out.